lib/vm.js

```diff
@@ -18,19 +18,21 @@ const realRunInContext = Script.prototype.runInContext;
 
 Script.prototype.runInThisContext = function(options) {
   if (options && options.breakOnSigint) {
-    return sigintHandlersWrap(() => {
+    const sigintHandler = () => {
```

Comment:
where is the doc you are referring to , i want to understand this a little better